.github/workflows/release.yml

```diff
@@ -147,8 +152,18 @@ jobs:
         run: ci/install.sh
       - name: wait
         run: sleep 600
+      - name: before downgrade test
+        run: ci/test.sh downgrade-setup
+        if: ${{ matrix.downgrade == true }}
+      - name: downgrade
+        run: ci/downgrade.sh
+        if: ${{ matrix.downgrade == true }}
+      - name: after downgrade test
+        run: ci/test.sh downgrade-check
+        if: ${{ matrix.downgrade == true }}
```

Comment:
Since in downgrade you go from one version to another, that's why i think it's necessary to run test after the downgrade.
what do you think?

Reply:
Yeah, I agree (that is what I was trying to say 😅, I think I wasn't clear) only test after the downgrade and not before, since before it will be tested in the `ci (aws)` job.

Reply:
sorry got it now, yeah i agree before test actually not necessary

Reply:
I'm changing v2 to have the same behavior 👷‍♂️

Reply:
addressed it